test/CMakeLists.txt

```diff
@@ -1062,31 +1063,50 @@ ecbuild_add_test( TARGET  test_iodaconv_atms_nc4
 
+if( iodaconv_eccodes_ENABLED )
+
 #==============================================================================
 # Generic NRT Ingest tests
 #==============================================================================
 
-ecbuild_add_test(TARGET  test_${PROJECT_NAME}_generic_bufr_raob
-                 TYPE    SCRIPT
-                 ENVIRONMENT "PYTHONPATH=${IODACONV_PYTHONPATH}"
-                 COMMAND bash
-                 ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
-                         netcdf
-                         "${Python3_EXECUTABLE} ${CMAKE_BINARY_DIR}/bin/decode_bufr_LDM_raob.py
-                         -i testinput/sonde_wmo_double.bufr
-                         -o testrun/wmo_raob_double.nc4"
-                         wmo_raob_double.nc4 ${IODA_CONV_COMP_TOL_ZERO})
-
-ecbuild_add_test(TARGET  test_${PROJECT_NAME}_generic_bufr_raob_multi_files
-                 TYPE    SCRIPT
-                 ENVIRONMENT "PYTHONPATH=${IODACONV_PYTHONPATH}"
-                 COMMAND bash
-                 ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
-                         netcdf
-                         "${Python3_EXECUTABLE} ${CMAKE_BINARY_DIR}/bin/decode_bufr_LDM_raob.py
-                         -i testinput/sonde_wmo_double.bufr testinput/sonde_wmo_double.bufr
-                         -o testrun/wmo_raob_quadruple.nc4"
-                         wmo_raob_quadruple.nc4 ${IODA_CONV_COMP_TOL_ZERO})
+  ecbuild_add_test(TARGET  test_${PROJECT_NAME}_generic_bufr_raob
+                   TYPE    SCRIPT
+                   ENVIRONMENT "PYTHONPATH=${IODACONV_PYTHONPATH}"
+                   COMMAND bash
+                   ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
+                           netcdf
+                           "${Python3_EXECUTABLE} ${CMAKE_BINARY_DIR}/bin/decode_bufr_LDM_raob.py
+                           -i testinput/sonde_wmo_double.bufr
+                           -o testrun/wmo_raob_double.nc4"
+                           wmo_raob_double.nc4 ${IODA_CONV_COMP_TOL_ZERO})
+
+  ecbuild_add_test(TARGET  test_${PROJECT_NAME}_generic_bufr_raob_multi_files
+                   TYPE    SCRIPT
+                   ENVIRONMENT "PYTHONPATH=${IODACONV_PYTHONPATH}"
+                   COMMAND bash
+                   ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
+                           netcdf
+                           "${Python3_EXECUTABLE} ${CMAKE_BINARY_DIR}/bin/decode_bufr_LDM_raob.py
+                           -i testinput/sonde_wmo_double.bufr testinput/sonde_wmo_double.bufr
+                           -o testrun/wmo_raob_quadruple.nc4"
+                           wmo_raob_quadruple.nc4 ${IODA_CONV_COMP_TOL_ZERO})
+
+#===============================================================================
+# Conventional data, aircraft Obs - AMDAR BUFR (WMO) converter
+#===============================================================================
+
+  ecbuild_add_test( TARGET  test_${PROJECT_NAME}_amdar
```

Comment:
CI testing defaults to `USE_ECCODES=OFF` so this test is not getting run. I tried `USE_ECCODES=ON` on my mac and I get this error:
```
Start testing: Mar 03 13:53 MST
----------------------------------------------------------
591/695 Testing: test_iodaconv_amdar
591/695 Test: test_iodaconv_amdar
Command: "/usr/local/bin/bash" "/Users/steveherbener/projects/DEVELOP/ioda-bundle/build/bin/iodaconv_comp.sh" "netcdf" "/usr/local/Frameworks/Python.framework/Versions/3.9/bin/python3.9 /Users/steveherbener/projects/DEVELOP/ioda-bundle/build/bin/amdar_bufr2ioda.py
                            -i testinput/amdar_wmo_multi.bufr
                            -o testrun/amdar_wmo_multi2.nc" "amdar_wmo_multi2.nc" "0.0"
Directory: /Users/steveherbener/projects/DEVELOP/ioda-bundle/build/iodaconv/test
"test_iodaconv_amdar" start time: Mar 03 13:53 MST
Output:
----------------------------------------------------------
DIFFER : VALUES OF GLOBAL ATTRIBUTE : source_files : "testinput/amdar_wmo_multi.bufr" <> "/work/noaa/da/gthompsn/src/ioda-bundle/iodaconv/test/testinput/amdar_wmo_multi.bufr"
<end of output>
Test time =   1.02 sec
----------------------------------------------------------
Test Failed.
"test_iodaconv_amdar" end time: Mar 03 13:53 MST
"test_iodaconv_amdar" time elapsed: 00:00:01
----------------------------------------------------------

End testing: Mar 03 13:53 MST
```

The global attribute "source_files" doesn't match and I think the fix is to update the test reference file.